Examples/Image/Detection/FastRCNN/BrainScript/PARAMETERS.py

```diff
@@ -56,6 +56,10 @@ def __init__(self, datasetName):
         self.cntk_max_epochs = -1                # set per data set below
         self.cntk_momentum_time_constant = -1    # set per data set below
 
+        # for Distributed learner
+        self.num_quantization_bits = 1   # set for distributed learner 
+        self.warm_up = 1                 # set for distributed learner
```

Comment:
warm_up = 1 seems too small for 1-bit SGD. For 32-bit data-parallel SGD you can set it to zero though.

Reply:
Are there any reference as to distributed learning? I didn't find the detailed specification on it.